modules/uhttp/client/client_middleware_test.go

```diff
@@ -73,8 +72,12 @@ func TestExecutionChainOutboundMiddlewareError(t *testing.T) {
 	assert.Nil(t, resp)
 }
 
-func withOpentracingSetup(t *testing.T, registerFunc auth.RegisterFunc, fn func(tracer opentracing.Tracer)) {
-	tracer, closer, err := tracing.InitGlobalTracer(&jconfig.Configuration{}, "Test", zap.NewNop(), metrics.NopCachedStatsReporter)
+func withOpentracingSetup(
+	t *testing.T, registerFunc auth.RegisterFunc, fn func(tracer opentracing.Tracer),
```

Comment:
nit: this is a new formatting strategy I haven't seen before :)

can we stick to one of the existing styles? for something that takes this many arguments, probably one-per-line like so:

```go
fung withOpenTracingSetup(
  *t *testing.T,
  registerFunc auth.RegisterFunc,
  // etc...
)
```